cmd/juju/commands/bootstrap_clouds.go

```diff
@@ -45,7 +45,9 @@ func formatCloudDetailsTabular(clouds cloudList, credStore jujuclient.Credential
 		for _, name := range cloudNames {
 			cred, err := credStore.CredentialForCloud(name)
 			if err != nil && !errors.IsNotFound(err) {
-				return errors.Annotatef(err, "error loading credential for cloud %q", name)
+				ctx.Warningf("error loading credential for cloud %v: %v", name, err)
+				p(name, "ERROR", "")
```

Comment:
I am no sure that we need to have both the error printed as ctx.Warningf as well as a record in the tabular output. ctx.Warningf  is enough I think...

It might be worthwhile adding tests where several credentials will have troubles loading in different order to make sure the output is as expected.

Reply:
I guess in this case it makes sense not to list the failed cloud, because then it isn't really "available" to bootstrap to.